src/Alert/Alert.stories.tsx

```diff
@@ -1,4 +1,5 @@
-import { Link, Typography } from "@material-ui/core";
+import Link from "@material-ui/core/Link";
+import Typography from "@material-ui/core/Typography";
```

Comment:
That's unnecessary, stories don't end up in bundle so there is no need for deep importing

Reply:
Eslint indicates warning here without change...